tests/integration/base.py

```diff
@@ -150,15 +151,40 @@ def databricks_cluster_profile(self):
             },
             'test': {
                 'outputs': {
-                    'dbsql': {
+                    'cluster': {
                         'type': 'databricks',
                         'host': os.getenv('DBT_DATABRICKS_HOST_NAME'),
                         'http_path': os.getenv('DBT_DATABRICKS_CLUSTER_HTTP_PATH'),
                         'token': os.getenv('DBT_DATABRICKS_TOKEN'),
                         'schema': self.unique_schema()
                     },
                 },
-                'target': 'dbsql'
+                'target': 'cluster'
+            }
+        }
+
+    def databricks_uc_cluster_profile(self):
```

Comment:
I wonder if we can combine the UC cluster profile with the regular cluster profile by simply giving it a cluster HTTP path that points it to a UC-enabled cluster (with initial catalog set to different values).